code/__DEFINES/preferences.dm

```diff
@@ -25,6 +25,7 @@
 #define MEMBER_PUBLIC	8192
 #define CHAT_NO_ADMINLOGS 16384
 #define DONATOR_PUBLIC	32768
+#define DISABLE_KARMA_TEXT_REMINDER	65536
```

Comment:
65536 = 1024 * 64 = 2**10 * 2**6 = 2**16
This is bit 17/32 (on the premise that BYOND uses 32-bit integers)